Src/library/wel/windows/wel_window.e

```diff
@@ -784,7 +785,8 @@ feature -- Status setting
 				end
 			end
 			swp_const := swp_const | swp_nomove | swp_nosize | swp_framechanged | swp_noactivate
-			l_success := {WEL_API}.set_window_pos (item, hwnd_const, 0, 0, 0, 0, swp_const)
+			l_dpi := cwin_get_dpi_for_window (item)
+			l_success := {WEL_API}.set_window_pos (item, hwnd_const, cwin_mul_div (0, l_dpi, 96), cwin_mul_div (0, l_dpi, 96), cwin_mul_div (0, l_dpi, 96), cwin_mul_div (0, l_dpi, 96), swp_const)
```

Comment:
`cwin_mul_div (0, l_dpi, 96)` always gives 0, so there is no reason to add these calls.